Pipeline/RunIDZCodeReview/RunCodeReview.groovy

```diff
@@ -0,0 +1,269 @@
+import com.ibm.dbb.repository.*
```

Comment:
You should add `@groovy.transform.BaseScript com.ibm.dbb.groovy.ScriptLoader baseScript` as the first line of the Groovy script.  This provides script caching which is useful on ZD&T.  This will also allow you to use the inherited method `getScriptDir()` which you calculate yourself on line 210.

Reply:
I have added the baseScript definition